k8s_handle/__init__.py

```diff
@@ -83,57 +205,17 @@ def main():
         log.warning("Explicit true/false arguments to --sync-mode and --dry-run keys are deprecated "
                     "and will be discontinued in the future. Use these keys without arguments instead.")
 
-    args = vars(args)
-    kubeconfig_namespace = None
-    settings.CHECK_STATUS_TRIES = args.get('tries')
-    settings.CHECK_DAEMONSET_STATUS_TRIES = args.get('tries')
-    settings.CHECK_STATUS_TIMEOUT = args.get('retry_delay')
-    settings.CHECK_DAEMONSET_STATUS_TIMEOUT = args.get('retry_delay')
-    settings.GET_ENVIRON_STRICT = args.get('strict')
-    settings.ONLY_TAGS = args.get('tags')
```

Comment:
No need to make additional global variables instead of passing them normally. 
Implicit dependencies are no better than explicit ones, and I hope sometime to erase the entire file settings, I think it's not needed in fact.